reference/constraints/UniqueEntity.rst

```diff
@@ -94,11 +95,11 @@ your user table:
         // src/AppBundle/Entity/Author.php
         namespace AppBundle\Entity;
 
-        use Symfony\Component\Validator\Constraints as Assert;
-
         // DON'T forget this use statement!!!
         use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 
```

Comment:
I know this "use" is a bit special because of the caution message ... but I'd remove the blank line and I'd show all "use" grouped. If you think that's not clear enough, we can reword:

`DON'T forget this use statement!!!`

as:

`DON'T forget the following use statement!!!`

Reply:
I thought the same and wanted to find a better approach as is, will try your proposal